R/zchunk_batch_modeltime.xml.R

```diff
@@ -0,0 +1,36 @@
+#' module_modeltime_batch_modeltime.xml
+#'
+#' Construct XML data structure for \code{modeltime.xml}.
```

Comment:
So, the idea is that XML conversion happens in specialized XML chunks, correct?  And every module that produces tables of data destined for XML outputs will have a corresponding XML chunk?